lvx backend/config.env

```diff
@@ -0,0 +1,5 @@
+NODE_ENV=development
```

Comment:
`.env` file should be added in `.gitignore` also, you should remove it from here, it contains sensitive details.